templates/common/aws/units/aws-kubelet-nodename.service.yaml

```diff
@@ -3,12 +3,19 @@ enabled: true
 contents: |
   [Unit]
   Description=Fetch kubelet node name from AWS Metadata
+
+  # Run afterburn service for collect info from metadata server
+  # see: https://coreos.github.io/afterburn/usage/attributes/
+  Requires=afterburn.service
+  After=afterburn.service
+
   # Wait for NetworkManager to report it's online
   After=NetworkManager-wait-online.service
   # Run before kubelet
   Before=kubelet.service
 
   [Service]
+  EnvironmentFile=/run/metadata/afterburn
```

Comment:
OK sorry I was wrong, this is documented https://coreos.github.io/afterburn/usage/attributes/